src/scripts/eastern_kingdoms/eastern_plaguelands/naxxramas/boss_four_horsemen.cpp

```diff
@@ -347,13 +349,19 @@ struct boss_four_horsemen_shared : public ScriptedAI
         {
             if ((DoCastSpellIfCan(m_creature, SPELL_SHIELDWALL)) == CAST_OK)
                 m_bShieldWall1 = false;
+                m_uiShieldWallTimer = 0;
```

Comment:
This line is indented as if it's part of the if statement, but there are no brackets around it so it's not part of it and will always be executed regardless if the shield wall cast succeeds. Change it to clarify your intention. If the result of DoCastSpellIfCan does not matter, then change the indentation to make it clear it's not a mistake and you did **not** intend it to be executed only if the check above is true.

Reply:
In theory it won't make a difference, since the first shield wall does not check for the timer. Even if failed ShieldWall1 cast set the timer to 0 , the boss will recast ShieldWall1 and set the timer to 0 again soon after.

Nevertheless, I will include it in the if statement to be neater and more organized overall.
Thank you!